cmd/rhoas/main.go

```diff
@@ -94,12 +95,33 @@ func generateDocumentation(rootCommand *cobra.Command) {
 	}
 }
 
-func initConfig(f *factory.Factory) {
+func initConfig(f *factory.Factory, logger logging.Logger) {
+	rhoasCfgDir, err := config.DefaultDir()
+	if err != nil {
+		logger.Errorf("Error fetching config dir.", err)
```

Comment:
nit: would it be better to return the error from `initConfig` and log it in `main`? Saves having to pass the logger.